Atomic/atomic.py

```diff
@@ -553,17 +553,17 @@ def cmd_env(self):
         os.environ['NAME'] = self.name or ""
         os.environ['IMAGE'] = self.image or ""
 
-        if hasattr(self.args, 'opt1') and self.args.opt1:
+        if self.args.get('opt1', False):
             os.environ['OPT1'] = self.args.opt1
```

Comment:
Yes good point, I would love to drop most of the opt handling, since we are now passing the environment from the atomic command into the containers runtime.